frontend/packages/dev-console/integration-tests/features/Knative/event-Sources.feature

```diff
@@ -10,13 +10,6 @@ Feature: Event Sources actions
         And user has selected namespace "aut-create-knative-event-source"
 
 
-    @manual
-    Scenario: Scroll bar on the Event Sources cards page
-        Given user is at the Event Sources page
-        And user has added more than 10 Event Sources
-        Then user will see the Scroll bar added to see all the card
-
-
     @manual
     Scenario: Filter the Event Sources
         Given user is at the Event Sources page
```

Comment:
there is no eventSource page if contextual now based on selection from catalog flow

Reply:
Updated